README.md

```diff
@@ -1 +1,135 @@
-# Math-magician-React
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
```

Comment:
- [ ] You have submitted a project, that has the template logo of Microverse, if your project has a logo, replace it with it, if it does not, then remove Microverse's own and leave it empty.